.github/workflows/notifications.yml

```
@@ -3,7 +3,7 @@ name: Send notifications
 on: [ release, issues, issue_comment, watch, pull_request_target ]
 
 jobs:
-  notification:
+notification:
```

actionlint (1.7.4)

unexpected key "notification" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)